nights/2024-05-04-README.md

```diff
@@ -1,39 +1,20 @@
 # 2024-05-04
 
-## Reviewed by:   your_name_here
+## Reviewed by: Jenna Martodam
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2024-05-04
 
 ## Unusual images?
-
-Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
-
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
-
-Delete this list if there are no unusual images.
++ `ey-uma-S001-R001-C016-rp.fit` -- nice satellite or plane track in here.
```

Comment:
`ey-uma-S001-R001-C018-rp.fit` through `ey-uma-S001-R002-C007-rp.fit` has clouds present.

Reply:
also please add "closes #718 " to the pull request description.